src/css/profile/mobile/common/listview.less

```diff
@@ -25,13 +25,34 @@ tau-expandable {
 	 * because this widget has canvas which has to be visible outside the listview
 	 */
 	// overflow: hidden;
-	margin: 0;
+	margin: auto;
+
+	// List width for Tablet
+	@media (orientation: portrait) and (min-width: 673px) {
```

Comment:
you need a branch by height.

* : If the height exceeds 411dp while being included in the width range, the corresponding value should be applied. / If the height is 411dp or less, the margin principle should be applied to w100%.